src/colorizer/utils/math_utils.ts

```diff
@@ -24,3 +22,47 @@ export function numberToStringDecimal(
 export function setMaxDecimalPrecision(input: number, decimalPlaces: number): number {
   return Number.parseFloat(numberToStringDecimal(input, decimalPlaces, true));
 }
+
+// Adapted from https://gist.github.com/ArneS/2ecfbe4a9d7072ac56c0.
+function digitToUnicodeSupercript(n: number): string {
+  const subst = [0x2070, 185, 178, 179, 0x2074, 0x2075, 0x2076, 0x2077, 0x2078, 0x2079];
```

Comment:
This is how the pros do it.  I would have just used "1e-20" and be done with it.